web/src/logs.types.ts

```diff
@@ -55,3 +56,21 @@ export type QueryRangeResponse<T = MatrixResult | StreamsResult> = {
     };
   };
 };
+
+export type RulesResponse = {
+  status: string;
+  data: {
+    groups: Array<{
+      rules: Array<{
+        labels: Record<string, number>;
+        annotations: Record<string, number>;
+        activeAt: string;
+        state: string;
+        value: string;
+      }>;
+      annotations: Record<string, number>;
+      labels: Record<string, number>;
+      query: string;
```

Comment:
Is this correct? Shouldn't `annotations` and `labels` only be within the rule object or the alert object?

Similarly for `query`, which I think should be `expr` on the `rule` object?

If we are not actually using these attributes, maybe better to remove them from the type anyway.

Reply:
good catch, the type was duplicating some fields based on https://prometheus.io/docs/prometheus/latest/querying/api/#rules. I fixed it to include the labels and query which are required for the detail page to query the correct tenant.